stock_request_picking_type/views/stock_request_order_views.xml

```diff
@@ -0,0 +1,83 @@
+<odoo>
+    <record id="view_stock_request_order_filter" model="ir.ui.view">
```

Comment:
I'm wondering why this view is defined here and not in base module ???

Reply:
I deleted this search view and improve the search view of the core.